sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/AzureTable.java

```diff
@@ -0,0 +1,23 @@
+package com.azure.data.tables;
+
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+
+public class AzureTable {
```

Comment:
We have checkstyles that require you to document public facing methods. you will have to add documentation to these or the build won't pass.

Reply:
Do you use AzureTable and AzureTableClient? It looks like you use TableServiceClient and TableClient. ? Perhaps these files can be deleted.